src/docs/product/data-management-settings/server-side-sampling/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Server-side Sampling
+sidebar_order: 1
+description: "Learn about how you can control which transactions Sentry retains."
+---
+
+Server-side sampling lets you control what transactions Sentry retains by setting sampling rules and rates so you see more of the transactions you want to explore further, and less of the ones you don’t, without reconfiguring the SDK or redeploying your application.
+
+![The Server-side Sampling page with a list of rules](server-side-sampling.png)
+
+Because every project is different, you can set multiple sample rules with different sample rates per project. For example, you may need more events from high converting pages, critical API endpoints, or need to focus on latency issues from your latest release.
+
+Server-side sampling also makes your SDK and server-side sampling largely independent of each other. This helps if you're building native applications with many releases active at a given time, or you've already configured server-side sampling to target specific transactions, but the SDK blocks more of those transactions being sent to our servers.
+
+## How Sentry Samples
+
+Let's imagine for a moment that the total number of transactions the SDK observes is a bucket of rocks. The way Sentry samples transactions is similar to a person sifting rocks: by pouring the bucket of rocks into a sieve, you're allowing rocks small enough to pass through it:
```

Comment:
I think when we talk to the total of things, we have to refer to traces instead of transactions here as the 1000 is traces. 

```suggestion
Let's imagine for a moment that the total number of traces the SDK observes is a bucket of rocks. The way Sentry samples traces is similar to a person sifting rocks: by pouring the bucket of rocks into a sieve, you're allowing rocks small enough to pass through it:
```

Reply:
Yes, I agree that in this case it might be better to refer to traces, rather than transactions.

Reply:
Hmmm, not sure this isn't going to be more confusing without updating the UI. The feature in the app primarily talks about setting transaction sampling rates, not traces sampling rates. I think we might be introducing complexity in a way that doesn't actually improve comprehension, if that makes sense.